conans/assets/templates/new_v2_cmake.py

```diff
@@ -59,6 +59,9 @@ class {package_name}TestConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     generators = "CMakeDeps", "CMakeToolchain"
 
+    def requirements(self):
```

Comment:
I don't love the fact that it is not possible to do ``requires = something``, and this is still not a possibility for python_requires. But even with that, it still sounds better to be explicit and inject only the reference and not magically the requires.